x-pack/plugins/profiling/public/components/bar_chart.tsx

```diff
@@ -21,10 +21,6 @@ export interface BarChartProps {
 }
 
 export const BarChart: React.FC<BarChartProps> = ({ id, name, height, data, x, y }) => {
-  useEffect(() => {
-    console.log(new Date().toISOString(), 'updated bar-chart');
```

Comment:
I removed a bunch of console.log statements which are causing lint errors. I can also disable the lint rule or extract them into a logger service, but thought it was perhaps a good opportunity to clean some things up.

Reply:
Sounds good. These statements were useful for quick-and-dirty benchmarking early in our development, but I think using the logger service or RUM would be ideal now.